torch/cuda/__init__.py

```diff
@@ -79,13 +79,24 @@ def _is_compiled() -> bool:
     r"""Returns true if compile with CUDA support."""
     return hasattr(torch._C, '_cuda_getDeviceCount')
 
+def _nvml_based_avail() -> bool:
+    return os.getenv('PYTORCH_NVML_BASED_CUDA_CHK') == '1'
```

Comment:
is the missing "e" in "check" a typo or intentional for abbreviation?

Reply:
I had intentionally abbreviated CHECK to CHK because the var seemed to be getting a bit long already but I'm happy to change it to CHECK if you guys think that's better. Just lmk...